infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/ServerExposer.java

```diff
@@ -178,6 +181,11 @@ public void expose(String namePrefix, Map<String, ? extends ServerConfig> server
     return exposedPorts;
   }
 
+  private boolean isTlsEnabled() {
+    String isTlsEnabled = System.getenv("CHE_INFRA_OPENSHIFT_ENABLE__TLS");
```

Comment:
-1 to use property in such a way. In this case, property added though property file won't work. Can you inject it instead?

Reply:
ok. lemme try to handle DI better